librz/core/cmd/cmd_search_rop.c

```diff
@@ -445,7 +444,7 @@ static RzRopConstraint *rop_constraint_parse_args(RzCore *core, char *token) {
  *
  * This function parses a list of arguments into a RzPVector of RzRopConstraint objects.
  */
-RZ_API RzPVector /*<RzRopConstraint *>*/ *rop_constraint_map_parse(RZ_NONNULL RzCore *core, const int argc, const char **argv) {
+RZ_API RZ_NULLABLE RzPVector /*<RzRopConstraint *>*/ *rop_constraint_map_parse(const RZ_NONNULL RzCore *core, const int argc, const char **argv) {
```

Comment:
You forgot ownership here. Also, I recommend removing all those `RZ_NULLABLE` from return values as I mentioned before.

Reply:
So the pointer ownership has to be explicitly marked for all the functions that returns a Pointer? Shouldn't it be `RZ_OWN` unless otherwise specified? I'm little bit confused on marking the ownership for `_new*` and other functions which return a pointer @XVilka

Reply:
Yes, please add those missing markings.